packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -31,14 +39,20 @@ function InputDateTimePicker(props) {
 							id={`${props.id}-date-picker`}
 							onChange={onDateChange}
 							dateFormat={props.dateFormat}
+							useUTC={props.useUTC}
+							timezone={props.timezone}
 							value={date}
+							hideTimezone
 						/>
 						<InputTimePicker
 							id={`${props.id}-time-picker`}
 							onChange={onTimeChange}
 							useSeconds={props.useSeconds}
 							value={time}
 						/>
+						{props.timezone && (
+							<TimeZone timezone={props.timezone} />
+						)}
```

Comment:
Related to my previous comment: shouldn't it be rendered on InputTimePicker then ?
In the current situation, we have the timezone info only in DateTime picker, but if we want to use a Time picker alone, it should have the info.
I suggest to move it.